test/unit/specs/core/injectExecuteCommand.spec.js

```diff
@@ -210,9 +210,12 @@ describe("injectExecuteCommand", () => {
   });
 
   it("executes the core commands", () => {
+    const componentRegistry = {
+      getCommand() {}
+    };
     const configureCommand = jasmine
       .createSpy()
-      .and.returnValue(Promise.resolve("configureResult"));
+      .and.returnValue(Promise.resolve(componentRegistry));
```

Comment:
This test was erroneously passing previously because this mock was resolving the promise with a string rather than an object. We have logic in Alloy that says if the command provides an object, return it, but if the command provides something else, replace it with an empty object: https://github.com/adobe/alloy/blob/c43cdd3be1637f8671f9cf660c1da71249d23494/src/core/injectExecuteCommand.js#L107.

We needed to more closely align our mock with reality in order for the faulty behavior to be exhibited.